static/admin/css/autocomplete.css

```diff
@@ -12,19 +12,25 @@ select.admin-autocomplete {
     padding: 0;
 }
 
-.select2-container--admin-autocomplete.select2-container--focus .select2-selection,
-.select2-container--admin-autocomplete.select2-container--open .select2-selection {
+.select2-container--admin-autocomplete.select2-container--focus
```

Comment:
Can you give more context on where all the css and js files & changes are coming from?

Reply:
im actually not sure what that is but all of our css looks the same